### PR TITLE
Translated content update

### PR DESCRIPTION
**Pull request by YisHub**
### Description
The content is being updated to be the same as the English version. The Spanish version was very outdated and no longer worked in Django 4

### Motivation
When I tried to take the course, I encountered several problems when trying to do it with the latest version of Django. When I reviewed the English version, I realized that the Spanish version was quite outdated.

By updating these changes, the experience of taking the course will be significantly improved. This is because the errors will have been fixed, which will eliminate the need to investigate why they occurred. Additionally, the improvements from the English version will be implemented, which will further expand the quality and content of the course

### CODE REVIEW

File: files/es/learn/server-side/django/home_page/index.md [hunk 1]
```diff
@@ -87,27 +87,27 @@ Abra **urls.py** y pegue la línea en negrita que aparece a continuación.
 
 ```python
 urlpatterns = [
-    url(r'^$', views.index, name='index'),
+    path('', views.index, name='index'),
 ]
 ```
 
-Esta función `url()` define un patrón URL (`r'^$'`), y una función vista que será llamada si el patrón es detectado (`views.index` — una función llamada `index()` en **views.py**). El patrón URL es una [expresión regular de Python](https://docs.python.org/3/library/re.html) (ER). Hablaremos un poco más sobre ERs más adelante en este tutorial, pero para este caso todo lo que necesitas saber es que en una ER de ^$ el patrón coincidirá con una cadena vacía (^ es un marcador de inicio de cadena y $ es un marcador de fin de cadena).
+Esta función `path()` define una cadena vacía (`''`), y una función vista que será llamada si el patrón es detectado (`views.index` — una función llamada `index()` en **views.py**). Hablaremos un poco más sobre los patrones URL más adelante en este tutorial, pero para este caso todo lo que necesitas saber es que en un patron de '' coincidirá con una cadena vacía.
```
Comment (Graywolf9) [inline files/es/learn/server-side/django/home_page/index.md:94]:
```suggestion
Esta función `path()` define una cadena vacía (`''`), y una función vista que será llamada si el patrón es detectado (`views.index` — una función llamada `index()` en **views.py**). Hablaremos un poco más sobre los patrones URL más adelante en este tutorial, pero para este caso todo lo que necesitas saber es que en un patron de `''` coincidirá con una cadena vacía.
```

**Review by Graywolf9 — APPROVED** [review]
Thank you so much for all your time and effort, this deffinetly would help to the `es` readers.

There's only one note: we put each file in separate PR to make easy and fast to review, also it helps that one file does not interrump the merge of the others, by this time I'm goint to merge it like that

Again, Thank you so mucho for all your time and 

Welcome to MDN

Feel free to keep contributing

**Comment by YisHub** [comment]
Hi Graywolf9,

Thank you so much for your comments. I am very glad that my translation can help Spanish readers. Your feedback is very important to me, and I will use it to improve my translations in the future.

I would also like to thank you for the welcome. I feel honored to be a part of this project and I hope to be able to contribute to it in a meaningful way.

Best regards,
YisHub

**Comment by Graywolf9** [comment]
Hi!

It's an honor for us to have you here, thank you again for for help and we are happy to see your new contributions!

**Pull request merged**